README.md

```diff
@@ -544,6 +557,44 @@ Please replace the followings variables (in addition to the general variables):
 - `$GIT_USERNAME` with the git username if the repository is access protected. Leave empty, when no authentication is required.
 - `$GIT_PASSWORD` with the git password if the repository is access protected. Leave empty, when no authentication is required. The password gets cleaned after cloning and is not accessible in the RDP connection.
 
+### Backup image
+
+Please run the following command:
```

Comment:
```suggestion
Please run the following command to backup from T4C to Git:
```

It's more clear if the purpose of the command is stated as well, since people do not tend to read top-to-bottom.